packages/tools/src/utilities/getMatchingViewport.ts

```diff
@@ -0,0 +1,29 @@
+import { getEnabledElements, utilities as csUtils } from '@cornerstonejs/core';
+import type { Annotation } from '../types';
+
+const { isEqual } = csUtils;
+
+/**
+ * Finds a matching viewport in terms of the orientation of the annotation data
+ * and the frame of reference.  This doesn't mean the annotation IS being displayed
+ * in the viewport, just that it could be by navigating the slice, and/or pan/zoom,
+ * without changing the orientation.
+ *
+ * @param annotation - to find a viewport that it could display in
+ * @returns The viewport to display in
+ */
+export default function getMatchingViewport(annotation: Annotation) {
```

Comment:
cornerstoneTools.utilities.getMatchingViewport is too general, can we say `getMatchingViewportForAnnotation` or `getViewportForAnnotation` or `getAnnotationViewport`.